jellyfin.ruleset

```diff
@@ -43,14 +45,18 @@
     <Rule Id="CA1812" Action="Info" />
     <!-- disable warning CA1822: Member does not access instance data and can be marked as static -->
     <Rule Id="CA1822" Action="Info" />
+    <!-- disable warning CA2000: Dispose objects before losing scope -->
+    <Rule Id="CA2000" Action="Info" />
 
     <!-- disable warning CA1054: Change the type of parameter url from string to System.Uri -->
     <Rule Id="CA1054" Action="None" />
+    <!-- disable warning CA1055: URI return values should not be strings -->
+    <Rule Id="CA1055" Action="None" />
+    <!-- disable warning CA1056: URI properties should not be strings -->
+    <Rule Id="CA1056" Action="None" />
```

Comment:
why disable these?

Reply:
It complains when you have a string property that contains the word "url"